src/main/java/io/jenkins/plugins/orka/client/ResponseBase.java

```diff
@@ -28,7 +30,15 @@ public String getMessage() {
         return this.errors.clone();
     }
 
-    public boolean hasErrors() {
+    public boolean isSuccessful() {
+        return this.httpResponse != null ? this.httpResponse.getIsSuccessful() : !this.hasErrors();
+    }
+
+    public String getErrorMessage() {
+        return this.hasErrors() ? Arrays.toString(this.getErrors()) : httpResponse.getBody();
```

Comment:
Why do we need to return `response's body` when there is no errors ? Shouldn't just return an empty string ?

Reply:
Because I do not want to rely on the API returning a response with a specific format.
If the API returns a response with a msg and not errors property, we will not display that.

Reply:
After discussing it in Slack, we decided to return `httpResponse.getBody()` only if we have error http status code.